src/Store/VimStoreConnector.re

```diff
@@ -744,6 +744,17 @@ let start =
         | _ => ()
         };
 
+        // Set configured line comment
+        editorBuffer
+        |> OptionEx.flatMap(Core.Buffer.getFileType)
+        |> OptionEx.flatMap(
+             Ext.LanguageConfigurationLoader.get_opt(languageConfigLoader),
+           )
+        |> OptionEx.flatMap((config: Ext.LanguageConfiguration.t) =>
+             config.lineComment
+           )
+        |> Option.iter(Vim.Options.setLineComment);
+
```

Comment:
This is in `synchronizeEffect`. Not sure if that's the best place to put it, but it's not a particularly expensive operation so this is probably fine.

Reply:
Seems like the best spot for now 👍 I'd like to eventually revisit the 'synchronization' and instead 'push' values into libvim... but for now, this is the right spot.